diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -14,6 +14,7 @@
 cc = meson.get_compiler('c')
 
 m_dep = cc.find_library('m')
+thread_dep = dependency('threads')
 dl_dep = cc.find_library('dl', required : false)
 egl_dep = cc.find_library('EGL')
 gst_codecs_deps = dependency('gstreamer-codecparsers-1.0')
@@ -44,6 +45,7 @@
         egl_dep,
         m_dep,
         dl_dep,
+        thread_dep,
     ],
     install: true,
     install_dir: get_option('libdir') / 'dri',
